app/consumer-democracy/app.go

```diff
@@ -633,6 +639,23 @@ func New(
 				fromVM[moduleName] = eachModule.ConsensusVersion()
 			}
 
+			// TODO: should have a way to read from current node home
```

Comment:
Should this code be added for both a normal democ consumer and a democ consumer that was previously sovereign? See https://github.com/cosmos/interchain-security/issues/783

Reply:
resolved in #783